compiler/rustc_hir_analysis/src/outlives/explicit.rs

```diff
@@ -55,6 +55,7 @@ impl<'tcx> ExplicitPredicatesMap<'tcx> {
                     ty::PredicateKind::Clause(ty::Clause::Trait(..))
                     | ty::PredicateKind::Clause(ty::Clause::Projection(..))
                     | ty::PredicateKind::WellFormed(..)
+                    | ty::PredicateKind::AliasEq(..)
```

Comment:
should also be possible to make this a bug. Though hopefully we should soon change this function to iterate over `Clause` instead of `PredicateKind`, making that irrelevant